### PR TITLE
lib/linked_vocabs: update for RDF 2.0

### LINKED ISSUE #16: incompatible with ruby-rdf 2.0
The `require "rdf/cli/vocab-loader"` line in https://github.com/projecthydra-labs/linked_vocabs/blob/master/lib/linked_vocabs.rb#L3 is causing an error, since that was removed from the RDF gem last March: https://github.com/ruby-rdf/rdf/commit/47b2774285ad69bed2f92d42a386dda354d7282b

### PR DESCRIPTION
**Pull request by dunn**
Closes #16.

### CODE REVIEW

File: lib/rdf/vocab/dcmitype.rb [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# -*- encoding: utf-8 -*-
+# frozen_string_literal: true
+# This file generated automatically using rdf vocabulary format from
```
Comment (dunn) [inline lib/rdf/vocab/dcmitype.rb]: Do we want these checked into git, or should I add `lib/rdf/vocab/*` to the `.gitignore`?

**Comment by dunn** [comment]
Thanks!  This is a breaking change, but you're still on `0.x.x`, so should this be `0.4.0`?

**Comment by no-reply** [comment]
Maybe we should just cut 1.0? I don't think it's likely we'll see a lot of churn on this in the future.

**Comment by dunn** [comment]
I'll let you decide!

**Pull request merged**